docs/reference/query-dsl/prefix-query.asciidoc

```diff
@@ -64,4 +64,12 @@ GET /_search
 You can speed up prefix queries using the <<index-prefixes,`index_prefixes`>>
 mapping parameter. If enabled, {es} indexes prefixes between 2 and 5
 characters in a separate field. This lets {es} run prefix queries more
-efficiently at the cost of a larger index.
\ No newline at end of file
+efficiently at the cost of a larger index.
+
+[[prefix-query-allow-expensive-queries]]
+===== Allow expensive queries
+Prefix queries will not be executed if <<query-dsl-allow-expensive-queries, `search.allow_expensive_queries`>>
```

Comment:
However, if <<index-prefixes, `index_prefixes`>> are enabled, an optimised query is built which is not considered slow, and will be executed in spite of this setting.